src/services/handleTodoFilter.ts

```diff
@@ -0,0 +1,21 @@
+import { Todo } from '../types/Todo';
+
+export function handleTodoFilter(
+  todoFromServer: Todo[],
+  filterBy: string,
+  title = '',
+): Todo[] {
+  const filteredTodo = todoFromServer
+    .filter(todo => todo.title.toLowerCase().includes(title.toLowerCase()));
+
+  switch (filterBy) {
+    case 'active':
```

Comment:
please create enum for `active`, `completed`, `all`

create a folder `enums` for it